scripts/remote_diff/show_config_hash.py

```diff
@@ -0,0 +1,29 @@
+import sys
```

Comment:
Please add a command explaining the purpose of this script and an example of its intended use.